tests/contrib/langchain/conftest.py

```diff
@@ -18,34 +18,6 @@ def ddtrace_config_langchain():
     return {}
 
 
-@pytest.fixture(scope="session")
```

Comment:
note: got rid of metrics and logs test. some were flaky for some reason, and seeing as we might take them out entirely, i just dropped all their tests.